webpack.config.js

```diff
@@ -10,9 +12,40 @@ const HtmlWebpackInjectPreload = require('@principalstudio/html-webpack-inject-p
 let ogImageUrl = process.env.RIOT_OG_IMAGE_URL;
 if (!ogImageUrl) ogImageUrl = 'https://app.element.io/themes/element/img/logos/opengraph.png';
 
-const additionalPlugins = [
-    // This is where you can put your customisation replacements.
-];
+const cssThemes = {
```

Comment:
How would this work if someone was to fork the project and create a new theme?
Would they need to tweak this config on top of changing their `.env` ?

Reply:
@gsouquet they would have to change this config anyways, so that doesn't really change much. In terms of development on only one, custom theme, then yes, they'd have to update the `.env` too.